singolo/index.html

```diff
@@ -0,0 +1,221 @@
+        </nav>
+      </div>
+      <div class="line-header"></div>
+    </header>
+    <section id="slider" class="slider">
+      <div class="wrapper">
+        <div id="slides" class="slides">
+          <div class="slide">
+            <div class="slide-1">
+              <div class="phone-vertical slide-2-pv">
+                <div class="phone-vertical__camera slide-2-pc"></div>
+                <div class="phone-vertical__speaker slide-2-psp"></div>
+                <div class="phone-vertical__screen slide-2-psc"></div>
+                <div class="phone-vertical__home-button slide-2-phb">
+                  <div
+                    class="phone-vertical__home-button_backlight slide-2-phbb"
+                  ></div>
+                </div>
+              </div>
+
+              <div class="phone-horizontal">
+                <div class="phone-horizontal__camera"></div>
+                <div class="phone-horizontal__speaker"></div>
+                <div class="phone-horizontal__screen"></div>
+                <div class="phone-horizontal__home-button">
+                  <div class="phone-horizontal__home-button_backlight"></div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="slide">
+            <div class="slide-2">
+              <div class="phone-vertical-left">
+                <div class="phone-vertical-left__camera"></div>
+                <div class="phone-vertical-left__speaker"></div>
+                <div class="phone-vertical-left__screen"></div>
+                <div class="phone-vertical-left__home-button">
+                  <div class="phone-vertical-left__home-button_backlight"></div>
+                </div>
+              </div>
+              <div class="phone-vertical-center">
+                <div class="phone-vertical-center__camera"></div>
+                <div class="phone-vertical-center__speaker"></div>
+                <div class="phone-vertical-center__screen"></div>
+                <div class="phone-vertical-center__home-button">
+                  <div
+                    class="phone-vertical-center__home-button_backlight"
+                  ></div>
+                </div>
+              </div>
+              <div class="phone-vertical-right">
+                <div class="phone-vertical-right__camera"></div>
+                <div class="phone-vertical-right__speaker"></div>
+                <div class="phone-vertical-right__screen"></div>
+                <div class="phone-vertical-right__home-button">
+                  <div
+                    class="phone-vertical-right__home-button_backlight"
+                  ></div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="buttons">
+        <div id="prev" class="slider-content__arrow-left"></div>
+        <div id="next" class="slider-content__arrow-right"></div>
+      </div>
+      <div class="line-slider"></div>
```

Comment:
the same here
![image](https://user-images.githubusercontent.com/11545672/104485350-6f64ae00-55db-11eb-994e-f93f44ba4d6b.png)

Reply:
implemented so the underlining, taken from the layout